cborg-json/src/Codec/CBOR/JSON.hs

```diff
@@ -44,24 +46,33 @@ decodeValue :: Bool -> Decoder s Value
 decodeValue lenient = do
     tkty <- peekTokenType
     case tkty of
-      TypeUInt    -> decodeNumberIntegral
-      TypeUInt64  -> decodeNumberIntegral
-      TypeNInt    -> decodeNumberIntegral
-      TypeNInt64  -> decodeNumberIntegral
-      TypeInteger -> decodeNumberIntegral
-      TypeFloat16 -> decodeNumberFloat16
-      TypeFloat32 -> decodeNumberFloating
-      TypeFloat64 -> decodeNumberFloating
-      TypeBool    -> Bool   <$> decodeBool
-      TypeNull    -> Null   <$  decodeNull
-      TypeString  -> String <$> decodeString
-
+      TypeUInt         -> decodeNumberIntegral
+      TypeUInt64       -> decodeNumberIntegral
+      TypeNInt         -> decodeNumberIntegral
+      TypeNInt64       -> decodeNumberIntegral
+      TypeInteger      -> decodeNumberIntegral
+      TypeFloat16      -> decodeNumberFloat16
+      TypeFloat32      -> decodeNumberFloating
+      TypeFloat64      -> decodeNumberFloating
+      TypeBool         -> Bool   <$> decodeBool
+      TypeBreak        -> Bool   <$> decodeBreakOr
+      TypeBytes        -> decodeByteString
+      TypeBytesIndef   -> decodeByteStringIndef []
+      TypeNull         -> Null   <$  decodeNull
+      TypeSimple       -> decodeSimple'
```

Comment:
Check the spec for how this is supposed to be converted. I'm not sure if it can be.